docs/get_started.ipynb

```diff
@@ -481,10 +511,166 @@
     {
```

Comment:
It looks like this image was broken before you made this change but it would be nice if you fixed it rather than just deleting it.
```suggestion
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "NVpnilhEgQXk"
      },
      "source": [
        "![Quickstart Gradient Tape](https://raw.githubusercontent.com/tensorflow/tensorboard/master/docs/images/quickstart_gradient_tape.png)"
      ]
    },
    {
```